Lib/mimetypes.py

```diff
@@ -542,6 +543,10 @@ def _default_mime_types():
         '.aiff'   : 'audio/x-aiff',
         '.ra'     : 'audio/x-pn-realaudio',
         '.wav'    : 'audio/x-wav',
+        '.otf'    : 'font/otf',
+        '.ttf'    : 'font/ttf',
+        '.woff'   : 'font/woff',
+        '.woff2'  : 'font/woff2',
```

Comment:
I verified from the RFC that these 4 lines and hence the corresponding test lines are correct.  There is no reference I see for the vnd-ms addition so cannot verify.  The merge conflict will be in the tests due to the switch to using subtests, which looks good to me.  So merge that first and fix this.

Reply:
Thanks for verifying.

`application/vnd.ms-fontobject` is vendor specific and can be seen for example at:

* https://www.iana.org/assignments/media-types/application/vnd.ms-fontobject
* https://developer.mozilla.org/en-US/docs/Web/HTTP/MIME_types/Common_types